README.rst

```diff
@@ -74,33 +67,29 @@ For example:
 
  CRYOLO_ENV_ACTIVATION = conda activate yourdesiredname
 
-crYOLO general model is not installed by default. You may install it by
-expanding the plugin in the plugin manager and install it.
-This will install the general model to a default location: ``~/scipion/software/em/cryolo_model-20190516``.
-
-If you wish to provide a different general model:
-Set **CRYOLO_GENERIC_MODEL** variable in the ``~/.config/scipion.conf`` variables section:
-
-For example:
-
-::
+Downloaded crYOLO and JANNI general models can be found, respectively, in the following locations:
 
- CRYOLO_GENERIC_MODEL = /your/desired/location/myownmodelname.h5
+``~/scipion/software/em/cryolo_model-[model_version]``.
```

Comment:
Scipion can be installed anywhere....so ~/scipion is misleading. I usually write <SCIPION_HOME>/software/em